judgels-backends/jophiel/jophiel-app/src/main/java/judgels/jophiel/user/UserResource.java

```diff
@@ -84,6 +86,15 @@ public User createUser(AuthHeader authHeader, UserData data) {
         return userStore.createUsers(data);
     }
 
+    @Override
+    @UnitOfWork
```

Comment:
```suggestion
    @UnitOfWork(readOnly = true)
```

(as this is non-mutating query and no locks are needed)